testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2011,12 +2011,8 @@ public void testCreateTableWithLongTableName()
         }
 
         String invalidTableName = validTableName + "z";
-        try {
-            assertUpdate("CREATE TABLE " + invalidTableName + " (a bigint)");
-        }
-        catch (Throwable e) {
-            verifyTableNameLengthFailurePermissible(e);
-        }
+        assertThatThrownBy(() -> assertUpdate("CREATE TABLE " + invalidTableName + " (a bigint)"))
```

Comment:
Long commit message - ` Ensure failure when creating invalid table name in testCreateTableWithLongTableName`.

Would `Verify failure for invalid table name in testCreateTableWithLongTableName` work?